session/session_test.go

```diff
@@ -3356,41 +3338,35 @@ PARTITION BY RANGE (c) (
 	result.Check(testkit.Rows("dc-1"))
 	// test local txn
 	tk.MustExec("insert into t1 (c) values (1)") // in dc-1 with dc-1 scope
-	result = tk.MustQuery("select * from t1")
-	c.Assert(len(result.Rows()), Equals, 4)
+	result = tk.MustQuery("select * from t1 where c < 100")
```

Comment:
Since no index is created for table `t1`, there are no test cases for `IndexReader`, `IndexLookup`, and `IndexMergeReader`.